Explorer/Assets/DCL/Navmap/NavmapController.cs

```diff
@@ -102,6 +107,19 @@ public class NavmapController : IMapActivityOwner, ISection, IDisposable
             lastParcelHovered = Vector2.zero;
         }
 
+        private void ToggleSection(bool isOn, TabSelectorView tabSelectorView, NavmapSections shownSection, bool animate)
+        {
+            if(isOn && animate && shownSection != lastShownSection)
+                sectionSelectorController.SetAnimationState(false, tabsBySections[lastShownSection]);
+
+            animationCts.SafeCancelAndDispose();

```

Comment:
Nice! I didnt know it had a SafeCancelAndDispose xDD

Reply:
:D